help/sequence-alignments.md

```diff
@@ -4,16 +4,31 @@ type: help
 categories: Website,help
 ---
 
-*Note: This article is currently out of date and corresponds to the interface of a previous version of the website*
+---
+title: Sequence alignments
+type: help
+categories: Website,help
```

Comment:
We're looking at adding/fixing help documents to all tools offered by UniProt. Should we add a new `tools` category?